ckan/new_authz.py

```diff
@@ -431,12 +431,13 @@ def auth_is_anon_user(context):
 
         See ckan/lib/base.py:232 for pylons context object logic
     '''
```

Comment:
@amercader, this change is a change for the better (avoiding pylons c object), but can you think of any reason not to do this? Do you happen to know of any cases where this would break anything?